utils/data/target_metadata/ullyses_aliases.csv

```diff
@@ -254,10 +254,10 @@ BP TAU,V-BP-TAU,BP TAU,V-BP-TAU,V BP TAU,HD 281934,2MASS J04191583+2906269,GAIA-
 CS CHA,V-CS-CHA,CS CHA,V-CS-CHA,V CS CHA,CHX 4,SSTC2D J110225.0-773336,2MASS J11022491-7733357,GAIA-DR2-5201179389431267584,SZ 9,CS CHA,,,,,,,,,,,,,
 CW TAU,V-CW-TAU,CW TAU,,V CW TAU,2MASS J04141700+2810578,GAIA-DR2-163184091252903936,CW-TAU,,,,,,,,,,,,,,,,
 DE TAU,V-DE-TAU,DE TAU,,V DE TAU,2MASS J04215563+2755060,GAIA-DR2-152511475478780416,,,,,,,,,,,,,,,,,
-DK TAU,V-DK-TAU,DK TAU,,V DK TAU,2MASS J04304425+2601244,DK TAU A,,,,,,,,,,,,,,,,,
+DK TAU,V-DK-TAU,DK TAU,,V DK TAU,2MASS J04304425+2601244,DK TAU A,V-DK-TAU-A,DK-TAU-A,,,,,,,,,,,,,,,
 DN TAU,V-DN-TAU,DN TAU,,V DN TAU,2MASS J04352737+2414589,GAIA-DR2-147606657186323712,HBC 65,,,,,,,,,,,,,,,,
 FM TAU,V-FM-TAU,FM TAU,,V FM TAU,2MASS J04141358+2812492,GAIA-DR2-163184366130809472,HBC 23,,,,,,,,,,,,,,,,
-HN TAU,V-HN-TAU,HN TAU,,V HN TAU,2MASS J04333935+1751523,GAIA-DR2-3314237734735803392,HN TAU A,HN-TAU,,,,,,,,,,,,,,,
+HN TAU,V-HN-TAU,HN TAU,,V HN TAU,2MASS J04333935+1751523,GAIA-DR2-3314237734735803392,HN TAU A,HN-TAU,V* HN Tau,,,,,,,,,,,,,,

```

Comment:
@jduval82 "V* HN Tau" needs to be changed to "V* HN TAU"

Reply:
I just pushed the change @Rplesha